docs/python_support/quickstart.md

```diff
@@ -0,0 +1,222 @@
+# Python support in OVMS - quickstart {#ovms_docs_python_support_quickstart}
+
+OpenVINO Model Server allows users to write custom processing nodes in Python, so they may have full control over what happens with the data reaching such node and what comes out of it.
+
+In this quickstart you'll create a servable with a single custom Python code that will expect a string and return the same string, but all in uppercase. 
+
+To acheive that let's follow the steps:
```

Comment:
In sphinx, it's possible to create links that will follow you to specific part of the document. We already do that in some docs. Can we use that here?

Reply:
This may appear on the side menu like for main quickstart:
https://docs.openvino.ai/2023.2/ovms_docs_quick_start_guide.html#doxid-ovms-docs-quick-start-guide
I'll confirm that, but if it is, then I think it's obsolete